app/views/development_metrics/defect_escape_rate/_values_details.erb

```diff
@@ -18,7 +18,11 @@
         <h5>Bugs by environment:</h5>
         <% metric[:values].each do |metric| %>
           <div class="item-list">
+            <% if metric.first -%>
             <%= metric.first.to_s.capitalize %>:
+            <% else -%>
+              None:
+            <% end -%>
```

Comment:
```suggestion
            <%= metric.first.to_s.capitalize.presence || 'None:' %>:
```

Apart from that, I don't remember if we have presenters in this project, if we do have it might go there.

Reply:
We don't have presenters in this project yet.